SudokuBLL/src/test/java/pkgGame/SudokuTest.java

```diff
@@ -13,14 +14,23 @@
 public class SudokuTest {
 	
 	@Test
-	public void sudoku2arg_test() {
+	public void sudokuTwoArgConstructor_test() {
 		int size = 9;
 		
 		Sudoku s = new Sudoku(size, eGameDifficulty.EASY);
 		
-		assertTrue(s.isPartialSudoku());
+		assertTrue(s.isSudoku());
 		assertTrue(s.getiSize()==9);
-		s.PrintPuzzle();
+
+		Field fDifficulty = Sudoku.class.
+		            getDeclaredField("eGameDifficulty");
+
+		fDifficulty.setAccessible(true);
+		
+		eGameDifficulty difficulty = (eGameDifficulty) fDifficulty.get(s);
+		
+		assertTrue(difficulty.getiDifficulty() == eGameDifficulty.EASY.getiDifficulty());
```

Comment:
Yep, good stuff. Always good to test as many fields, parameters, or aspects of a class that you know are true.